lib/charms/loki_k8s/v0/loki_push_api.py

```diff
@@ -936,7 +936,7 @@ def _resolve_dir_against_charm_path(charm: CharmBase, *path_elements: str) -> st
 class NoRelationWithInterfaceFoundError(Exception):
     """No relations with the given interface are found in the charm meta."""
 
-    def __init__(self, charm: CharmBase, relation_interface: str = ""):
+    def __init__(self, charm: CharmBase, relation_interface: str = None):
```

Comment:
All mypy is asking us to do is
```suggestion
    def __init__(self, charm: CharmBase, relation_interface: Optional[str] = None):
```
which is indeed per standard. But not a must. Up to you.